vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/resources/META-INF/resources/frontend/vaadin-spreadsheet/vaadin-spreadsheet-styles.js

```diff
@@ -1055,6 +1054,347 @@ export const spreadsheetStyles = css`
   .clear-filters-button {
     width: 100%;
   }
+
+  /*
+   * Lumo theme variant styles
+   */
+  :host([theme~="lumo"]) {
+    background-color: var(--lumo-base-color);
+  }
+
+  :where(:host([theme~="lumo"])) .v-spreadsheet .sheet .cell {
+    font-family: var(--lumo-font-family);
+    color: var(--lumo-body-text-color);
+    background-color: var(--lumo-base-color);
+    font-size: var(--lump-font-size-s);
+    border-right: 1px solid var(--lumo-contrast-20pct);
+    border-bottom: 1px solid var(--lumo-contrast-20pct);
+    padding: 0 var(--lumo-space-xs);
+  } 
+
+  :host([theme~="lumo"]) .v-spreadsheet {
+    border: 1px solid var(--lumo-contrast-20pct);
+    font-family: var(--lumo-font-family);
+    font-size: var(--lumo-font-size-m);
+    color: var(--lumo-body-text-color);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar {
+    background-color: var(--lumo-base-color);
+    border-bottom: 1px solid var(--lumo-contrast-20pct);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .functionfield,
+  .v-spreadsheet .functionbar .addressfield {
+    font-size: var(--lumo-font-size-s);
+    padding-left: var(--lumo-space-xs);
+    color: var(--lumo-body-text-color);
+    background-color: unset;
+    padding: 0px 0px 0px var(--lumo-space-s);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .arrow {
+    font-size: var(--lumo-font-size-xs);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .fixed-left-panel {
+    border-right: 1px solid var(--lumo-contrast-20pct);
+    background-color: white;
+    background-image: linear-gradient(var(--lumo-contrast-5pct), var(--lumo-contrast-5pct));
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .adjusting-right-panel {
+    padding-right: var(--lumo-space-xs);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .addressfield {
+    background-color: white;
+    background-image: linear-gradient(var(--lumo-contrast-5pct), var(--lumo-contrast-5pct));
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .functionfield {
+    font: unset;
+  }
+
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .formulaoverlay {
+    color: var(--lumo-base-color);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .formulaoverlay span {
+    border-radius: var(--lumo-border-radius-s);
+  }
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .sheet .cell {
+    background-color: var(--lumo-base-color);
+    border-right: 1px solid var(--lumo-contrast-20pct);
+    border-bottom: 1px solid var(--lumo-contrast-20pct);
+    padding: 0px var(--lumo-space-xs);
+  }
```

Comment:
There's an extra closing curly bracket character and the next styling block seems to be duplicated (in fact, after removing the bracket, the custom formatting breaks again).